### ISSUE TITLE
FR: Expose block level in GraphQL

### ISSUE THREAD

**Issue by membla** [spicywebau/craft-neo#501]
In order to mutate blocks by id while keeping them in the same position it seems necessary to provide the `level` of the block, like:

```
mutation {
  save_slides_slide_Entry(id: "857", blockContent: {blocks: {richText: {id: "863", blockRichText: "Mutate me", level: 4}}}) {
    id
    title
    blockContent(level: 0) {
      __typename
      ... on blockContent_richText_BlockType {
        id
        fieldId
        ownerId
        blockRichText
        # level here?
      }
    }
  }
}
```
I am not sure if it would be possible to make the specification of `level` optional, but if not, could the block expose `level` so it is at hand for the mutation?

**Comment by membla** [comment]
https://github.com/spicywebau/craft-neo/blob/359e8f78362d0f336400bc3bb27aa09fb26e22fc/src/gql/types/input/Block.php#L101

Would simply defaulting to `null` here instead of `1` not let us just specify the id of the block to mutate, keeping it in its position? If that is the case that seems like the expected behavior when not providing a level IMO.

**Comment by ttempleton** [comment]
Just added the `level` field in Neo 2.11.13. Thanks also for reporting the bug with existing blocks being moved to level 1, it's only supposed to be new blocks that get level 1 as default, so that should be fixed in 2.11.13.
